product_docs/docs/pem/8.1.0/pem_admin/01_overview/02_installing_pem_overview.mdx

```diff
@@ -4,12 +4,12 @@ title: "Installing PEM - Overview"
 
 For detailed instructions about installing PEM, please consult the `PEM Installation Guides`, available at:
 
-   https://www.enterprisedb.com/docs/
+   <https://www.enterprisedb.com/docs/>
 
 The basic steps involved in the PEM installation process are:
 
-1. Install the PEM server components. The PEM server software and backend database (named pem) may reside on the same host as the supporting httpd server, or may reside on a separate host. The PEM server installer installs a PEM agent and the PEM Web client on the host of the PEM server.
+1.  Install the PEM server components. The PEM server software and backend database (named pem) may reside on the same host as the supporting httpd server, or may reside on a separate host. The PEM server installer installs a PEM agent and the PEM Web client on the host of the PEM server.
```

Comment:
I'm curious why there would be an extra space after the period. Is this something specific to the Markdown flavor we use?

Reply:
Short answer: puts the list at a 4-space indent. Longer answer: the "normalization" script I'm using for this enforces a consistent formatting for most things, even when Markdown / CommonMark / GFM is more flexible. Normally this is irrelevant, but comes into play when there are paragraphs within a list item as the indent level *can* make the difference between a paragraph being interpreted as a paragraph vs. a code block.